portal-files/Dockerfile

```diff
@@ -1,19 +1,31 @@
 FROM php:8.1-apache-bullseye
 
-# Install Debian packagess
+RUN mkdir -p /run/php/
```

Comment:
Required for FPM's PID tracking. If UNIX socket connection is used, the socket is also created here.